javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -137,6 +138,15 @@ public ProxyMethod map(Operation operation) {
             .findFirst()
             .orElse(null);
 
+        Set<String> responseContentTypes = operation.getResponses().stream()
+                .filter(r -> r.getProtocol() != null && r.getProtocol().getHttp() != null && r.getProtocol().getHttp().getMediaTypes() != null)
+                .flatMap(r -> r.getProtocol().getHttp().getMediaTypes().stream())
+                .filter(s -> !s.isEmpty())
+                .collect(Collectors.toSet());
+        if (!responseContentTypes.contains("application/json")) {
+            responseContentTypes.add("application/json;q=0.9");
```

Comment:
What is q=0.9 for?

Reply:
This is for operation that does not specify "application/json" for media type in response (for protection that some swagger missing "application/json"). It add a default "application/json" with less importance, so that if e.g. service want to respond with application/xml specified in swagger, it got default q=1 so it will go, if e.g. service forget to specify "application/json" in swagger, it still will have this as an option.

Reply:
If you prefer, I could move this to template part to handle.

Reply:
I think mapper should be the place for setting default values.